examples/demo_bgp.pp

```diff
@@ -333,89 +333,78 @@
   }
 
   # --------------------------------------------------------------------------#
-  # Configure Neighbor-level Address Family IPv4 Unicast
+  # Configure Neighbor-level Address Family IPv4 Unicast (default vrf)
   # --------------------------------------------------------------------------#
-  cisco_bgp_neighbor_af { '55.77 blue 1.1.1.1 ipv4 unicast':
+  if $operatingsystem == 'ios_xr' {
```

Comment:
I'd be tempted to remove the 'if' logic here and have these be set for both Nexus and XR. Nexus needs these configs too to have a 'working' config - it's just not as persnickety about validating the config as XR.